packages/features/borrow/src/borrow-fixed/hooks/use-totals-data.tsx

```diff
@@ -0,0 +1,57 @@
+import { FormattedMessage } from 'react-intl';
+import { TokenBalance } from '@notional-finance/core-entities';
+import {
+  useAllMarkets,
+  useSelectedNetwork,
+} from '@notional-finance/notionable-hooks';
+
+export const useTotalsData = (selectedDepositToken: string | undefined) => {
+  const network = useSelectedNetwork();
+  const data = useAllMarkets();
+  const {
+    yields: { fCashBorrow, liquidity },
+  } = data;
+  let tvlData: any = null;
```

Comment:
I don't think you need to use a `let` or an `any` type here.

`const tvlData = liquidity?.find(
      (data) => data.underlying?.symbol === selectedDepositToken
    )?.tvl;`

That will get you a correctly typed variable that may be undefined.